### PR TITLE
Timeline feature for arbitration

### PR DESCRIPTION
**Pull request by yathishram**
## Description:
A basic timeline to represent different stages of the arbitration.

### CODE REVIEW

File: src/components/Arbitrations/ArbitrationDetail.js [hunk 1]
```diff
@@ -42,21 +47,47 @@ const ArbitrationDetail = (props) => {
...
       } catch (err) {
-        console.log('ERROR', err)
+        console.log('ERROR', err);
         return false;
       }
     }
     load();
   }, []);
 
+  useEffect(() => {
```
Comment (koshikraj) [inline src/components/Arbitrations/ArbitrationDetail.js]: Can you revert this change and move this back to `arbDetails` component as there is a fix in the other branch?

File: src/components/Arbitrations/agreementDetails/arbDetails.js [hunk 2]
```diff
@@ -14,15 +14,12 @@ import { useHistory } from 'react-router-dom';
 import ProcedureStatementForm from '.././modals/ProcedureStatement';
 import StatementForm from '../modals/StatementForm';
 import ArbitrationCardDispute from '../../../assets/ArbitrationCardDispute.svg';
-import { getArbitrationDetails } from '../../../lib/contracts/SPC';
 
-function ArbDetails({ groupId, contractAddress, NODE, account, caller, parties }) {
+function ArbDetails({ groupId, contractAddress, details, caller, parties, account, loading }) {
```
Comment (koshikraj) [inline src/components/Arbitrations/agreementDetails/arbDetails.js]: Revert these changes.

**Comment by koshikraj** [comment]
We can test only the first stage of the timeline as of now, right?

File: src/components/Arbitrations/ArbitrationDetail.js (hunk 1 above)
Comment (yathishram) [reply src/components/Arbitrations/ArbitrationDetail.js]: Okay, the issue is that `getArbitrationDetails` returns the stage and time also. Now I'll be creating a new function in the contract for us to return the stage and time and later I'll be moving this to `arbDetails`

**Comment by yathishram** [comment]
Yeah, first stage works. When the response submitted is created it'll move to next stage.

**Comment by koshikraj** [comment]
LGTM.
